scriptmodules/emulators/fs-uae.sh

```diff
@@ -9,9 +9,11 @@
 # at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
 #
 
+FSUAE_EXTS=(.adf .ipf .dms .adz .zip)
+
 rp_module_id="fs-uae"
 rp_module_desc="Amiga emulator - FS-UAE integrates the most accurate Amiga emulation code available from WinUAE"
-rp_module_help="ROM Extension: .adf"
+rp_module_help="ROM Extension: ${FSUAE_EXTS[*]}\n\nCopy your Amiga games to $romdir/amiga\n\nCopy a required BIOS file (e.g. kick13.rom) to $biosdir"
```

Comment:
I think I would prefer just putting the extensions in platforms.cfg - the emulators themselves document what they support and we have this in other cases where some emulators don't support all formats.